application/core/ConsoleApplication.php

```diff
@@ -149,4 +149,14 @@ public function getPluginManager()
         return $this->getComponent('pluginManager');
     }
 
+    public function getAssetManager()
```

Comment:
Not sure these functions are needed, you should always be able to do: Yii::app()->assetManager, etc for other components.

Reply:
We are using the ExportSurveyResultsService class from Limesurvey. This class is calling some internal functions which requires the added methods.

Reply:
Hm, alright. Can you then please add docblocks? Like:

```
/**
 * @return CComponent
 */
 ```

Reply:
asset manager and script are for web construction, then surely not loaded in console.

https://github.com/LimeSurvey/LimeSurvey/blob/3f1f5f17c3d7dd0aa7885d1bcbad583772ae3ccf/framework/web/CWebApplication.php#L149
Nothink in https://github.com/LimeSurvey/LimeSurvey/blob/master/framework/console/CConsoleApplication.php